### DISCUSSION TITLE
How to save the outputs in a log file while running locally!

### DISCUSSION THREAD

**Discussion by gsaha009** [columnflow/columnflow#439] [Q&A]
Hi,

I was trying to save the output of say, `SelectEvents` task in a log-file, but I noticed that it is not working. For batch submission, I saw the log-files, but is there any way to make it work for the interactive jobs as well?

Thanks :)

**Comment by gsaha009** [comment]
Btw, I tried with `&>` and it worked as usual, but the in-built `--log-file` argument is not working properly

**Comment by pkausw** [comment]
Hi @gsaha009 ,

The remote workflows and their derivatives (e.g. HTCondor) originate from law, so this is not a columnflow feature by itself. Nevertheless, I think the short answer is that the `--log-file` argument is specific to the remote workflows, so I'm not sure that it's considered in local workflows at all. Maybe @riga can add something here.

That being said, you can of course always pipe the stdout output into a file, or just use the remote workflow if you really need a task-specific output at the end. In case this doesn't cover your needs, is there a specific use case that you have in mind?

**Reply by gsaha009** [reply]
Yes, I am using `&>` to log the local output. But, I would say it would be very nice if I can import the main `logger` that logs the `columnflow` tasks to add my local outputs as well, e.g. some `print` statements.
This is not urgent at all, but I thought it would be nice.
Thanks :)
